test/common/upstream/health_check_fuzz.cc

```diff
@@ -265,6 +512,18 @@ void HealthCheckFuzz::initializeAndReplay(test::common::upstream::HealthCheckTes
     replay(input);
     break;
   }
+  case envoy::config::core::v3::HealthCheck::kGrpcHealthCheck: {
+    type_ = HealthCheckFuzz::Type::GRPC;
+    grpc_fuzz_test_ = std::make_unique<GrpcHealthCheckFuzz>();
+    try {
```

Comment:
i wonder if the switch should be inside the try/catch rather than a try/catch within each switch.

Reply:
Sure